src/components/Feed/Feed.test.js

```diff
@@ -24,7 +23,7 @@ describe("Tests feed component", () => {
   });
 
   it("should render tweets list", () => {
-    const { tweets: { tweets, isLoading } } = store.getState();
+    const { tweets: { tweets } } = store.getState();
     
     const wrapper = renderer.create(
       <Provider store={store}>
```

Comment:
This provider is really needed? Why?

Reply:
oh, pardon, i thought i aswered up there. That is because of a nested component inside feed named <TweetConfirmDelete />, it dispatches an action to redux

Reply:
the component TweetConfirmDelete